### ISSUE TITLE
Outline effect pass fills entire screen after clearing selection

### ISSUE THREAD

**Issue by thmxv** [vanruesc/postprocessing#101]
When creating and outline effect and starting with an empty selection everything is displayed fine without any outline.

When selecting objects with selectObject() or setSelection() everything is still fine and the outline is visible.

After clearing the selection using clearSelection() or deselectObject() the effect pass fills the whole screen with the outline color.

**Comment by vanruesc** [comment]
Hi,

I'm unable to reproduce this issue in the [OutlineEffect Demo](https://vanruesc.github.io/postprocessing/public/demo/#outline); deselecting all objects seems to work as expected.

Could you please provide a code example that shows the issue? You can use this [JSFiddle](https://jsfiddle.net/8rmxa2t7/) as a base.

**Comment by thmxv** [comment]
I could not reproduce the issue exactly but something "somewhat" similar: https://jsfiddle.net/8rmxa2t7/2/

I moved the object creation/addition to the scene AFTER the composer/render passes setup (In my app they are added to the scene by callback and the scene content can change).

**Comment by thmxv** [comment]
Just to comment on why I cannot reproduce the issue exactly in JSFiddle is because my app is written in F# and quite complex.

**Comment by thmxv** [comment]
I managed to reproduce the "screen filling effect" https://jsfiddle.net/8rmxa2t7/5/
The change is deactivating blur.

So to resume:
- adding objects to the scene after creating the outline effect causes bug later when clearing the selection
- if using outline blur the bug is some kind of "persistent" outline that stays fixed when we rotate the view
- if not using blur it fills the screen with the outline color

**Comment by vanruesc** [comment]
Thanks for following up on this. I can see the issue now.

As a temporary workaround please add the following line directly after the instantiation of the `OutlineEffect` (even if there are no objects in the scene yet):

```js
outlineEffect.setSelection(scene.children);
```

I'll report back as soon as the problem has been fixed.

**Comment by vanruesc** [comment]
Hm, that's unfortunate. Thanks a lot for the information, though. It might take a while to get to the bottom of this.

**Comment by vanruesc** [comment]
I think I've found the issue. It appears that a clear color was incorrect.

If possible, please add the following line after the instantiation of the `OutlineEffect` and check if it works in your environment:

```js
outlineEffect.clearPass.clearColor.set(0x000000);
```

https://jsfiddle.net/8rmxa2t7/7/

**Comment by thmxv** [comment]
Yes, it fixes the issue for me. Thanks a lot.

**Comment by vanruesc** [comment]
Fixed in `postprocessing@5.3.2`.
